graphics/zbar/Portfile

```
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                zbar
+version             0.10
+categories          graphics
+platforms           darwin
+maintainers         nomaintainer
+license             GNU LGPL-2.1
```

Omit `GNU`, and append with `+` since in this case ZBar specifies "or any later version":

```suggestion
license             LGPL-2.1+
```